packages/manager/modules/pci/src/projects/project/quota/quota.html

```diff
@@ -1,286 +1,292 @@
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
 
-            <button
-                type="button"
-                class="oui-button oui-button_primary no-transition"
-                data-ng-if=":: !$ctrl.quotas || ($ctrl.quotas && $ctrl.quotas.length === 0)"
-                disabled
-            >
-                <span
-                    data-translate="pci_projects_project_quota_protect_more_btn"
-                ></span>
-                <span
-                    class="oui-icon oui-icon-external-link"
-                    aria-hidden="true"
-                ></span>
-                <span
-                    class="sr-only"
-                    data-translate="pci_projects_project_quota_open_new_tab"
-                ></span>
-            </button>
+    <div class="text-center" data-ng-if="$ctrl.isLoading">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div data-ng-if="!$ctrl.isLoading">
+        <!-- THERE IS NO PAYMENT MEAN - SHOW A MESSAGE TO ADD IT -->
+        <div
+            class="oui-box"
+            data-ng-if="$ctrl.isQuotaRestricted() && !$ctrl.hasDefaultPaymentMethod"
+        >
+            <p
+                class="font-weight-bold"
+                data-translate="pci_projects_project_quota_restricted_account"
+            ></p>
+            <p
+                data-translate="pci_projects_project_quota_restricted_account_unlock"
+                data-translate-values="{'url': $ctrl.paymentmeanUrl}"
+            ></p>
         </div>
 
-        <div data-ng-if="$ctrl.region === 'US'" data-write-right-required>
+        <!-- THERE IS A PAYMENT MEAN BUT QUOTA IS RESTRICTED -  -->
+        <div
+            class="oui-box"
+            data-ng-if="$ctrl.isQuotaRestricted() && $ctrl.hasDefaultPaymentMethod"
+        >
+            <p
+                class="font-weight-bold"
+                data-translate="pci_projects_project_quota_restricted_paymentmean_active"
+            ></p>
             <button
+                class="oui-button oui-button_primary"
                 type="button"
-                class="oui-button oui-button_primary no-transition"
-                data-ng-if="!$ctrl.quotas || ($ctrl.quotas && $ctrl.quotas.length === 0)"
-                data-ng-disabled="true"
-                data-translate="pci_projects_project_quota_protect_more_btn"
+                data-ng-click="$ctrl.unleashAccount()"
+                data-translate="pci_projects_project_quota_restricted_unlock_button"
             ></button>
+        </div>
+
+        <!-- ITS NOT RESTRICTED AND THERE IS A PAYMENT MEAN -->
+        <div class="oui-box" data-ng-if="!$ctrl.isQuotaRestricted()">
+            <p
+                class="font-weight-bold"
+                data-translate="pci_projects_project_quota_protect_explain"
+            ></p>
+            <p write-right-required>
+                {{ :: 'pci_projects_project_quota_protect_more' | translate }}
+            </p>
```

Comment:
```suggestion
            <p write-right-required data-translate="pci_projects_project_quota_protect_more">
            </p>
```
what is "write-right-required" ? is it a class for place the text to right alignement ?